.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+logs/
```

Comment:
The `.gitignore` file currently includes a line to ignore `logs/`, which aligns with the task requirement to ensure that log files are not pushed to GitHub.